### PR TITLE
Increase default cell width to fix layout issue on rendertron

### PR DESCRIPTION
**Pull request by nsharma123**
https://webarchive.jira.com/browse/WEBDEV-5266

### CODE REVIEW

File: src/infinite-scroller.ts [hunk 1]
```diff
@@ -380,9 +380,9 @@ export class InfiniteScroller
     const sentinelHeightCss = css`var(--infiniteScrollerSentinelDistanceFromEnd, 200rem)`;
     const rowGapSizeCss = css`var(--infiniteScrollerRowGap, 1.7rem)`;
     const colGapSizeCss = css`var(--infiniteScrollerColGap, 1.7rem)`;
-    const cellMinWidth = css`var(--infiniteScrollerCellMinWidth, 10rem)`;
+    const cellMinWidth = css`var(--infiniteScrollerCellMinWidth, 16rem)`;
```
Comment (nsharma123) [inline src/infinite-scroller.ts:383]: Taking the fallback cellMinWidth value from this reference which make it correct in inspect-element on https://archive.org/search?noscript=true&query=hello 
see attached:

Comment (nsharma123) [reply src/infinite-scroller.ts:383]: Screenshot from 2022-05-16 12-15-03

File: src/infinite-scroller.ts [hunk 2]
```diff
@@ -380,9 +380,9 @@ export class InfiniteScroller
...
     const cellMaxWidth = css`var(--infiniteScrollerCellMaxWidth, 1fr)`;
-    const cellMinHeight = css`var(--infiniteScrollerCellMinHeight, 10rem)`;
+    const cellMinHeight = css`var(--infiniteScrollerCellMinHeight, 22.5rem)`;
```
Comment (iisa) [inline src/infinite-scroller.ts:385]: @nsharma123 can we make an offshoot review app with changing the css var values from there?  that may not need a new version bump and just an offshoot mr...

File: src/infinite-scroller.ts (hunk 1 above)
Comment (iisa) [reply src/infinite-scroller.ts:383]: 👍

**Review by iisa — APPROVED** [review]
👍

File: package.json [hunk 3]
```diff
@@ -3,7 +3,7 @@
   "description": "An infinite scroller web component.",
   "license": "AGPL-3.0-only",
   "author": "Internet Archive",
-  "version": "0.1.2",
+  "version": "0.1.2-alpha.1",
```
Comment (iisa) [inline package.json:6]:
```suggestion
  "version": "0.1.2,
```
```suggestion
  "version": "0.1.2-alpha.1",
```

**Review by iisa — APPROVED** [review]
@nsharma123 let's get this in + cut an official tag for distribution.

**Review by iisa — APPROVED** [review]
@neeraj this is fine to merge and create official tag. please remove alpha tag from PR before merging. thanks!

**Pull request merged**